mousestyles/behavior/behavior_tree.py

```diff
@@ -110,9 +110,10 @@ def _display_tree(self, level=0):
             root = struct
             children = []
 
-        result = "\n" + " " * 5 * level + "{}: {} {}".format(root,
-                                                             self[root],
-                                                             self.units[root])
+        result = "\n" + " " * 5 * level + \
+            "{}: {:.6f} {}".format(root,
```

Comment:
@peter Should we make the number of decimal places an input parameter for the user?